dashboard/app/models/user_script.rb

```diff
@@ -19,9 +20,17 @@
 #
 
 class UserScript < ActiveRecord::Base
+  include SerializedProperties
+
   belongs_to :user
   belongs_to :script
 
+  validates :script, presence: true
```

Comment:
👍 

Might as well add this validation for `user` as well. 
Note they can be combined as `validates_presence_of :user, :script`

Reply:
oh right, I was scared about adding either of these validations. do we think there is a chance this will break things in prod, in a case where a validation in an existing codepath starts failing? if errors start turning up in honeybadger, I suppose we could revert just this validation change, and the corresponding unit test which will start failing without it: https://github.com/code-dot-org/code-dot-org/blob/608d34847f1a26ef11a96e6631a98ac83854df41/dashboard/test/controllers/user_scripts_controller_test.rb#L24

Reply:
It shouldn't break anything because these records can't be saved anyway. We'd get a Sql error for the null values. If some code path is actually expecting the sql error for some reason, then I say we change it.